signwriting_transcription/pose_to_signwriting/Online_bash_file.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# Clone the repository
+git clone https://github.com/sign-language-processing/signwriting-transcription.git
+cd signwriting-transcription
+
+# Install the required packages
+pip install .[dev,pose_to_signwriting]
+pip install ruamel.yaml
```

Comment:
should move this dependency to the pyproject file

Reply:
done